GoMeet/src/main/webapp/test/spec/specCreateMeetingStep1.js

```diff
@@ -0,0 +1,76 @@
+describe ("Add guest function", function() {
+
+  beforeAll(function() {
+    addGuest(document);
+  });
+
+  afterAll(function() {
+    removeGuest('guest-1', document); 
+  });
+
+  it ("Should add one li element to the html page", function() {
+    var listSize = document.getElementById('guest-list').childNodes.length; 
```

Comment:
nit: Google JavaScript style guide specifies to 'Declare all local variables with either const or let. The var keyword must not be used.'

Reply:
Fixed :)